controllers/util/provider.go

```diff
@@ -19,6 +19,10 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+const (
+	ProviderCertsMountPoint = "/mnt/cert"
+)
```

Comment:
What is the reason for moving this here? I believe this is only used from the provide server package so it should live there